connect/basic-auth-extension/src/main/java/org/apache/kafka/connect/rest/basic/auth/extension/BasicAuthSecurityRestExtension.java

```diff
@@ -80,7 +95,8 @@ public void close() throws IOException {
 
     @Override
     public void configure(Map<String, ?> configs) {
-
+        // If we failed to retrieve a JAAS configuration during startup, throw that exception now
+        CONFIGURATION.get();
```

Comment:
Could you add a test which confirms that we're propagating the exception here? At the moment, the test verifies that the wrapping method works, but doesn't verify that it's used by the rest extension during the loading phase.

Reply:
Ack, done.